pkg/envoy/cds/response.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 // NewResponse creates a new Cluster Discovery Response.
-func NewResponse(meshCatalog catalog.MeshCataloger, proxy *envoy.Proxy, _ *xds_discovery.DiscoveryRequest, _ *certificate.Manager, proxyRegistry *registry.ProxyRegistry) ([]types.Resource, error) {
+func NewResponse(meshCatalog catalog.MeshCataloger, proxy *envoy.Proxy, _ *xds_discovery.DiscoveryRequest, _ *certificate.Manager, _ *registry.ProxyRegistry) ([]types.Resource, error) {
```

Comment:
Are these unused args necessary to fulfill an interface?

Reply:
yes they are, there will be an opportunity to remove that though with the rest of the snapshotcache + refactoring that is currently ongoing